index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<!-- <a href="https://hokulani.org/">Click here to visit Hoku's website!</a> -->
+<a href="#">Home</a>
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="styles/style.css" rel="stylesheet">
+  <!-- <link href="styles/about.css" rel="stylesheet"> -->
+  
+  <title>Hoku's Portfolio 2022</title>
+</head>
+<!--body-->
+<body>
+  <!--header, main, footer -->
+  <header>
+    <h1>Hoku's Portfolio 2022</h1>
+    <nav>
+      <ul>
+        <li>
+          <a href="/pages/about.html">About</a>
+          <!-- <link href="styles/about.css" rel="stylesheet"> -->
+        <li>
+          <a href="/pages/portfolio.html">My work</a>
+        </li>
+       
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <p> Aloha E komo mai! My name is Hoku and I am so grateful to be here today!  
```

Comment:
You could also put a section tag around this paragraph tag. The section tag could have a class like "greeting"
```html
<section class="greeting">
    <p>
Aloha E komo mai! My name is Hoku and I am so grateful to be here today! Appreciate you taking the time to look at my portfolio!
    </p>
</section>